### PR TITLE
Updated from upstream

### PR DESCRIPTION
**Pull request by keanolane**

### CODE REVIEW

File: resources/custom/Homestead.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      type: "nfs"
+    - map: "{vendorPath}/../files/config"
+      to: /home/vagrant/config
+      type: "nfs"
+
+sites:
+    # API 3
+    - map: api3.test
+      to: /home/vagrant/code/api3/public
+      php: "7.1"
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml:28]: We should add API 5

File: resources/custom/Homestead.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - map: auth-service.test
+      to: /home/vagrant/code/auth-service/public
+      php: "7.1"
+
+    # Contractors Tool
+    - map: contractors-tool.test
+      to: /home/vagrant/code/contractors-tool/public
+      php: "7.1"
+
+    # Dashboard API
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml:45]: Is this in use?

File: resources/custom/Homestead.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    - map: dashboard-api.test
+      to: /home/vagrant/code/dashboard-api/public
+      php: "7.1"
+
+    # Dealerships Tool
+    - map: dealerships-tool.test
+      to: /home/vagrant/code/dealerships-tool/public
+      php: "7.1"
+
+    # Installs Reporting
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml]: This can be removed

File: resources/custom/Homestead.yaml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    - map: installs-reporting.test
+      to: /home/vagrant/code/installs-reporting/public
+      php: "7.1"
+
+    # Install tools - Ordering tools
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml]: I would change to be inline with the opsworks stacks - so `Installs Tool`

File: resources/custom/Homestead.yaml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    - map: installs-tool.test
+      to: /home/vagrant/code/installs-tool/public
+      php: "7.1"
+
+    # Install tools - Renault
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml]: I would change to be inline with the opsworks stacks - so `Renault`

File: resources/custom/Homestead.yaml [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    - map: ocpp-service.test
+      to: /home/vagrant/code/ocpp-service/public
+      php: "7.1"
+
+    # Pod Comms Service
+    - map: pod-comms-service.test
+      to: /home/vagrant/code/pod-comms-service/public
+      php: "7.1"
+
+    # POD Point - Web
```
Comment (mikefrancis) [inline resources/custom/Homestead.yaml:80]: This is the same as the one below it, think one can be removed?

**Review by mikefrancis — APPROVED** [review]

File: resources/custom/Homestead.yaml (hunk 2 above)
Comment (keanolane) [reply resources/custom/Homestead.yaml:45]: I personally use it. It's the simple dashboard I've created to grab and regroup the issues from Sentry by the most common ones

**Pull request merged**